include/matx/core/print.h

```diff
@@ -543,7 +561,40 @@ namespace matx {
 
     #ifdef __CUDACC__
       cudaDeviceSynchronize();
-      if constexpr (is_tensor_view_v<Op>) {
+      if constexpr (is_sparse_tensor_v<Op>) {
+        using Format = typename Op::Format;
+	index_t nse = op.Nse();
+        fprintf(fp, "nse    = %" MATX_INDEX_T_FMT "\n", nse);
+        fprintf(fp, "format = ");
+	Format::print();
+	//
+        // TODO: how to access pos/crd/val data properly in MatX style?
```

Comment:
Is POSData a matx tensor? Then operator() is correct

Reply:
Well, technically it is the .data() field of a Storage container, possibly taken from a matx vector (but one could also allocate their own buffer directly). Is () op still correct then?

Reply:
I think in general we should require a tensor to be passed in for those values and not a storage type. I suppose storage can be used if it's always 1D linear memory, but the tensor is more like the other APIs and would give us flexibility later.

Reply:
see above, we will need to only provide factory methods that take matx tensors